test/functional/wallet_dump.py

```diff
@@ -39,7 +39,7 @@ def read_dump(file_name, addrs, script_addrs, hd_master_addr_old):
                     if keytype == "inactivehdmaster=1":
```

Comment:
This one should be fixed up in commit
05dfe84ec2 [refactor] manually change remaining instances of master key to seed.

Reply:
done